src/AtelierTomato.Markov.Storage.Sqlite/Model/AuthorPermissionRow.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AtelierTomato.Markov.Model;
+using AtelierTomato.Markov.Model.ObjectOID.Parser;
+
+namespace AtelierTomato.Markov.Storage.Sqlite.Model
+{
+	public class AuthorPermissionRow
+	{
+		public string Author { get; set; }
+		public string QueryScope { get; set; }
+		public string AllowedScope { get; set; }
+		private readonly MultiParser<IObjectOID> ObjectOIDParser = new([new SpecialObjectOIDParser(), new BookObjectOIDParser(), new DiscordObjectOIDParser()]);

```

Comment:
no, why? this wastes objects for each data row coming from the DB.
maybe okay if static? but even then it's kinda yucky. we should look around what proper architecture would be.